src/NLog/Targets/NetworkLogEventDroppedReason.cs

```diff
@@ -49,6 +49,10 @@ public enum NetworkLogEventDroppedReason
         /// <summary>
         /// Discarded LogEvent because attempted to open more than <see cref="NetworkTarget.MaxConnections"/> connections
         /// </summary>
-        MaxConnectionsOverflow
+        MaxConnectionsOverflow,
+        /// <summary>
+        /// Discarded LogEvent because attempt to establish or write to network connection failed
+        /// </summary>
+        FailedToSend
```

Comment:
Can this be changed to `NetworkError` and change xml-description to `Discarded LogEvent because of network communication error`